dsperse/src/prove/prover.py

```
@@ -0,0 +1,311 @@
+    run_path = os.path.join(run_dir, latest_run)
+    run_path = os.path.join(run_path, "slice_0")
+
+    # Initialize prover
+    prover = Prover()
+
+    # Run proving
+    print(f"Proving run {latest_run} for model {base_paths[model_choice]}...")
+    results = prover.prove(run_path, slices_dir, backend="jstprove", tiles_range=[0, 1])
+
+    # Display results
+    print(f"\nProving completed!")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)